app/models/form/question.rb

```diff
@@ -70,9 +70,9 @@ def has_inferred_check_answers_value?(case_log)
 
   def update_answer_link_name(case_log)
     if type == "checkbox"
-      answer_options.keys.any? { |key| case_log[key] == "Yes" } ? "Change" : "Answer"
+      answer_options.keys.any? { |key| case_log[key] == "Yes" } ? "Change <span class=\"govuk-visually-hidden\">#{check_answer_label.to_s.downcase}</span>".html_safe : "Answer"
```

Comment:
We should also include the same visually hidden text for the ‘Answer’ variant of the link.

Reply:
Got it, will add it for the answer link as well

Reply:
@paulrobertlloyd I've made this change and have added the screenshots to the PR description 😄